blog/2024-08-22-3-ways-store-data-for-computer-vision-applications/index.mdx

```diff
@@ -4,8 +4,8 @@ description: "Learn how to store data in computer vision applications by using R
 authors: alexey
 tags: [tutorials, computer vision]
 slug: tutorials/computer-vision/3-ways-stora-data-for-computer-vision-applications
```

Comment:
typo

Reply:
I've kept it on purpose. too many changes need to be made to fix it and we lose the history of google search.

Reply:
All good, but it needs to be aligned with the redirect in staticwebapp.config.json, it was changed to a version without typo in this PR. It's also possible to add a 301 redirect to correct the typo and keep Google ranking.